gc/workercollector.go

```diff
@@ -26,7 +26,15 @@ func (wc *workerCollector) Run() error {
 	logger.Debug("start")
 	defer logger.Debug("done")
 
-	affected, err := wc.workerLifecycle.StallUnresponsiveWorkers()
+	affected, err := wc.workerLifecycle.DeleteUnresponsiveEphemeralWorkers()
+
```

Comment:
small nit: no extra space here; `err` assignment is always flush with its `err != nil` check